src/components/GetInviteKey/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { useQuery } from '@tanstack/react-query';
+import { getInviteKey } from 'apis/admin/manageGroup';
+
+import FlexContainer from 'components/@commons/FlexContainer';
+import SubmitButton from 'components/@commons/SubmitButton';
+import Text from 'components/@commons/Text';
+import React from 'react';
+import CopyToClipboard from 'react-copy-to-clipboard';
+import styled from 'styled-components';
+
+interface Props {
+  modalOffHandler: () => void;
+}
+
+const GetInviteKey = ({ modalOffHandler }: Props): JSX.Element => {
+  const { data: inviteKeyData } = useQuery(['inviteKey'], getInviteKey, { suspense: true });
+
+  return (
+    <FlexContainer $wFull $padding="20px" $gap="30px">
+      <FlexContainer $gap="10px">
+        <Text size="xl" weight="bold">
+          초대 링크
+        </Text>
+        <Text>아래 링크에 접속하면 그룹에 가입됩니다.</Text>
+      </FlexContainer>
+      <Box>
+        <Text>{inviteKeyData?.data.invitationKey}</Text>
+      </Box>
+      <FlexContainer $gap="10px">
+        <CopyToClipboard text={inviteKeyData?.data.invitationKey as string}>
```

Comment:
여기도 undefined인 경우를 대응하여 빈문자열을 넣어주시면 되겠습니다!
```suggestion
        <CopyToClipboard text={inviteKeyData?.data.invitationKey || ''}>
```